doc/api/crypto.md

```diff
@@ -2800,6 +2800,47 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt([min, ]max[, callback])`
+<!-- YAML
+added:
+  - CHANGEME
```

Comment:
```suggestion
  - REPLACEME
```

(see the earlier comment – `REPLACEME` is a special string, other values will not be picked up during the release process)